packages/uniforms-bootstrap4/src/components/fields/FormGroup.js

```diff
@@ -41,11 +42,13 @@ const FormGroup = ({
             <section className={classnames(wrapClassName, gridClassName(grid, 'input'))}>
                 {children}
                 {makeHelp(help, helpClassName)}
+                {error && showError ? makeHelp(error, 'text-help') : ''}
```

Comment:
Yes, I get it, but `error` is an object, not a `String` - `errorMessage` is a `String`.